src/Microsoft.AspNet.OData.Shared/Formatter/Deserialization/ODataResourceDeserializer.cs

```diff
@@ -492,7 +653,7 @@ private object ReadNestedResourceInline(ODataResourceWrapper resourceWrapper, IE
             {
                 Path = readContext.Path,
                 Model = readContext.Model,
-                Request = readContext.Request,
+                Request = readContext.Request
```

Comment:
```suggestion
                Request = readContext.Request,
```
So that in future updates addition won't show 2 changes.